server/server.go

```diff
@@ -111,6 +111,16 @@ type Server struct {
 	mapMu               sync.RWMutex
 	creatingCollections map[string]bool
 	iRODSConnections    map[string]int
+	iRodsTracker        SlackConfig
+}
+
+// SlackConfig is used for slack messages that debounce, such as clients
+// uploading and iRODS connections.
+type SlackConfig struct {
```

Comment:
This isn't a "SlackConfig", which would be config for slack. This is config for a message debouncer.